lib/utils/conf_gen.py

```diff
@@ -996,4 +1044,20 @@ async def generate(
         lines = []
 
         self._generate_helper(lines, list(intermediate_dict.values())[0])
-        return "\n".join(lines)
+        return "\n".join(lines).replace("{\n\n", "{\n")
```

Comment:
Why does this happen? Is it possible to fix it when `lines` is generated?

Reply:
fixed